config/initializers/active_fedora_env.rb

```diff
@@ -6,13 +6,15 @@ def initialize
   def load_fedora_config
     return @fedora_config unless @fedora_config.empty?
 
-    fedora_setting = Settings.fedora_url || ENV['FEDORA_URL']
+    fedora_setting = Settings.fedora.url || ENV['FEDORA_URL']
```

Comment:
Looks like this is failing in circleci.  Maybe best to safe navigate it?
```suggestion
    fedora_setting = Settings.fedora&.url || ENV['FEDORA_URL']
```